src/sliders.py

```diff
@@ -76,12 +76,15 @@ def return_of_adata_meta():
     @reactive.effect
     def random_sample():
         adata = _adata_qc.get()
+        if adata is None:
```

Comment:
It should be sufficient to check only once if `adata` is None

Reply:
okee

Reply:
done